content/jsoc/gsoc/graphs.md

```diff
@@ -5,51 +5,36 @@ title:  JuliaGraphs – Summer of Code
 
 # {{< get_param title >}}
 
+Thank you for your interest in supporting JuliaGraphs in the upcoming Summer of Code!
```

Comment:
Not sure this part is needed, it's a presentation page and people might not (yet) be interested in supporting lg

Reply:
They've navigated to this page, so they're (possibly) interested in supporting us. I don't think it's a bad idea to reinforce this feeling :)